qa/suites/upgrade/octopus-x/parallel/workload/test_rbd_api.yaml

```diff
@@ -1,6 +1,10 @@
 meta:
 - desc: |
    librbd C and C++ api tests
+overrides:
+  ceph:
+    log-ignorelist:
+      - is full \(reached quota
```

Comment:
Also, since RemoveFullTry test cases aren't present in octopus, `octopus-x` should be fine as is.

Reply:
Yes, that is correct I went on the assumption POOL_FULL would be present with other tests. It may be best to explicitly include `POOL_FULL` with the newly added exclude.

Reply:
@idryomov *-p2p suites live only in named branches, so in the example `quincy-p2p` it's in `quincy` only

Reply:
> Also, since RemoveFullTry test cases aren't present in octopus, `octopus-x` should be fine as is.

Nice catch! Done.